src/coreclr/System.Private.CoreLib/src/System/Reflection/Metadata/RuntimeTypeMetadataUpdateHandler.cs

```diff
@@ -11,11 +11,15 @@ namespace System.Reflection.Metadata
     /// <summary>Metadata update handler used to clear a Type's reflection cache in response to a metadata update notification.</summary>
     internal static class RuntimeTypeMetadataUpdateHandler
     {
+        public static bool HotReloadDeltaApplied { get; private set; }
```

Comment:
Could we make the name of this more generic and in line with the method that actually sets it, e.g. instead of `HotReloadDeltaApplied`, call it `CachesCleared` or something like that?

Reply:
Sure